src/v3/src/transformer/redirect/redirectTransformer.ts

```diff
@@ -76,7 +76,7 @@ export const redirectTransformer = (
 
   uischema.elements.unshift({
     type: 'Description',
-    options: { content: loc(titleContent, 'login', [contentParams]) },
+    options: { content: loc(titleContent, 'login', contentParams.length > 0 ? contentParams : undefined) },
```

Comment:
Might it be simpler to rewrite some of this logic so that all of the work is enclosed in each `if/else`? Sort of like this:

```
if (condition1) {
  unshift a description element matching condition 1 onto uischema
} else if (condition2) {
  unshift a description element matching condition 2 onto uischema
} else {
  unshift a default description element onto uischema
}
```

Reply:
If this works it would eliminate the `let titleContent` and also the `contentParams` issues